sdk/engine/include/NcEngine.h

```diff
@@ -36,39 +33,13 @@ namespace nc
 
             /**
              * @brief Get a pointer to the active registry.
-             * @return Registry*
              */
-            virtual auto Registry() noexcept -> nc::Registry* = 0;
+            virtual auto GetRegistry() noexcept -> Registry* = 0;
```

Comment:
We keep going back and forth between prefixing our getters with "Get". I thought we'd last decided to leave out the "Get". Why change now?